### DISCUSSION TITLE
Is there a mod for cycling throgh grouped taskbar items?

### DISCUSSION THREAD

**Discussion by p3P4** [ramensoftware/windhawk-mods#892] [General]
Is there a mod for cycling throgh grouped taskbar items like in 7+TT? If not - is it possible to create one?

**Comment by m417z** [comment]
There's no such mod, but it's planned. You can vote for it [here](https://ramensoftware.com/windhawk-mods-for-the-windows-11-taskbar) under "Left click on combined item: Cycle through windows".
